src/lit.js

```diff
@@ -1 +1 @@
-export * from 'https://unpkg.com/lit-html@^1.0.0/lit-html.js';
\ No newline at end of file
+export * from 'lit-html';
```

Comment:
this entire file should be deleted

i think this is an accidental vestige — i can't find it being imported anywhere in the codebase